sdk/LATEST

```diff
@@ -1,5 +1,5 @@
+d721044a70f3cf460c2522303aa6bd74266ed005 3.2.0-snapshot.20240807.13218.0.vd721044a SPLIT_RELEASE
 850097649f4f32f5316f517b677bd5440a8794ad 3.0.0-snapshot.20240708.12963.0.v85009764 SPLIT_RELEASE
-4ee22923e84d6630ba399cbe565d04bc872a5271 2.9.4-adhoc.20240729.12882.0.v4ee22923 SPLIT_RELEASE
```

Comment:
Is the removal of 2.9.4 on purpose?

Reply:
no, but we should clean adhoc snapshot once a while.
adhoc snapshots are snapshot of not merged commit.

Reply:
What is clear is that we need to update the RELEASE.md file.
I think the release.sh script could update and clean the LATEST file at the same time.
I will see with @paulbrauner-da what we can do.